markdown-format/README.md

```diff
@@ -59,6 +59,92 @@ It has multiple sentences.
 Let's format it!
 ```
 
+## Integration with Formatting Tools
+
+### treefmt
+
+[treefmt](https://github.com/numtide/treefmt) is a universal code formatter that runs multiple formatters with one command.
+
+To integrate markdown-format with treefmt, you'll need to create a wrapper script since markdown-format outputs to stdout rather than formatting files in place.
+
+1. Create a wrapper script (e.g., `markdown-format-inplace.sh`):
```

Comment:
markdown-format should support this (inplace mode) out of the box, script should not be needed